cmd/deploy/deploy.go

```diff
@@ -400,7 +401,6 @@ func (dc *DeployCommand) RunDeploy(ctx context.Context, deployOptions *Options)
 	oktetoLog.DisableMasking()
 	oktetoLog.SetStage("done")
 	oktetoLog.AddToBuffer(oktetoLog.InfoLevel, "EOF")
-	oktetoLog.SetStage("")
```

Comment:
In case of error, we don't clear the stage. This implies that the log printed by `main.go` would be under `done` stage, but any log under that stage is ignored, which is what we want in this case. It is not an idea solution, but for now, it is the best to fix the issue until we improve all the logging